admin-manual/installation/installation.rst

```diff
@@ -234,45 +237,71 @@ two different sources of packages.
    sudo apt-get update
    sudo apt-get upgrade
 
-4. Install the storage service package
+5. Install Elasticsearch
+
+.. code:: bash
+
+   sudo apt-get install elasticsearch
+
+6. Install the storage service package
+
+.. code:: bash
+
+   sudo apt-get install -y --force-yes archivematica-storage-service
```

Comment:
Yes, there are new packages built from stable/1.6.x branch signed with the production key `5236CA08`